src/sparse_merkle_tree/smt.rs

```diff
@@ -2538,13 +2887,12 @@ mod tests {
             1u8, 2, 3, 4, 1, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71, 3, 3, 71,
             3, 3, 71, 3, 3, 71, 3,
         ]);
-        let res = SparseMerkleTree::verify(
+        let res = SparseMerkleTree::verify_and_prepare_proof_map(
```

Comment:
Why is this change needed in this unit test and not also in others (for example in `test_mid_proof()`)?